syscall/zsyscall_windows.go

```diff
@@ -18,7 +18,7 @@ var (
 )
 
 func GetModuleFileName(hmodule uint32, buf *uint16, buflen uint32) (n uint32, err error) {
-	r0, _, e1 := syscall.Syscall(procGetModuleFileNameW.Addr(), 3, uintptr(hmodule), uintptr(unsafe.Pointer(buf)), uintptr(buflen))
+	r0, _, e1 := syscall.Syscall(procGetModuleFileNameW.Addr(), 3, uintptr(hmodule), uintptr(unsafe.Pointer(buf)), uintptr(buflen)) // #nosec
```

Comment:
Actually ideally it would be just this file. (Or I guess files in syscall that start with 'z' which appears to be the convention for the tool that generates this stuff.)